### PR TITLE
swaylock: add page

### PR DESCRIPTION
**Pull request by rafay99-epic**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/swaylock.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# swaylock
+
+> Swaylock is a screen locking utility for Wayland compositors.
```
Comment (kbdharun) [inline pages/linux/swaylock.md:3]:
```suggestion
> Screen locking utility for Wayland compositors.
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have a suggestion for this page.

File: pages/linux/swaylock.md (hunk 1 above)
Comment (rafay99-epic) [reply pages/linux/swaylock.md:3]: Sure, happy to make some changes brother.

**Comment by rafay99-epic** [comment]
Done with the changes brother.

**Review by einverne — APPROVED** [review]
Looks good to me after @acuteenvy 's suggestion.

File: pages/linux/swaylock.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# swaylock
+
+> Screen locking utility for Wayland compositors.
+> More information: <https://www.mankier.com/1/swaylock>.
```
[the remark this answers is not in the document]
Comment (rafay99-epic) [reply pages/linux/swaylock.md:4]: Sure, I will make some change and then I will let you know.

**Comment by rafay99-epic** [comment]
Alright Brother, I am done with all the changes as requested.

**Review by einverne — APPROVED** [review]

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**